src/renderer/pages/Status.tsx

```diff
@@ -39,7 +40,28 @@ const Status: React.FC = () => {
           </div>
         </div>
         <div className='block is-flex is-justify-content-center mt-6'>
-          <Link to='/recording' className='button is-light has-text-danger'>Go to recording</Link>
+          <button
+            className='button is-warning'
+            disabled={!(deviceConnected(state.left) || deviceConnected(state.right) && !leadIntegrityPending)}
+            onClick={() => {
+              [state.left, state.right].forEach(async item => {
+                const { name } = item
+                setLeadIntegrityPending(true)
+
+                if (!deviceConnected(item)) { return }
+
+                try {
+                  dispatch({ type: ActionType.IntegrityTest, name }) // NOP
+                  const response = await (window as any).deviceManagerService.integrityTest({ name, leadList: integrityTestPairs() })
+                  dispatch({ type: ActionType.IntegrityTestSuccess, name }) // NOP
+                } catch (e) {
+                  dispatch({ type: ActionType.IntegrityTestFailure, message: e.message, name })
+                }
+
+                setLeadIntegrityPending(false)
```

Comment:
Does this re-enable the the button briefly in between the left and right tests? Do you want that?

Reply:
Fixed